packages/conf-mingw-w64-zlib-i686/conf-mingw-w64-zlib-i686.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+synopsis: "Virtual package for zlib on i686 mingw-w64 (32-bit x86)"
+description: "Ensures the i686 version of zlib for the mingw-w64 project is available"
+maintainer: "David Allsopp <david@tarides.com>"
+license: "zlib"
+homepage: "http://www.zlib.net/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+flags: conf
+available: os = "win32"
+build: ["pkgconf" "--personality=i686-w64-mingw32" {os-distribution = "cygwin"} "zlib"]
+depends: [
+  "msys2" {build & os = "win32" & os-distribution = "msys2"}
+  "msys2-mingw32" {os = "win32" & os-distribution = "msys2"}
+  "conf-pkg-config" {build}
+  "conf-mingw-w64-gcc-i686" {build}
+]
+depexts: [
+  ["mingw64-i686-zlib"] {os = "win32" & os-distribution = "cygwin"}
+  ["mingw-w64-i686-zlib"] {os = "win32" & os-distribution = "msys2"}
+]
```

Comment:
However, if I'm supposed to maintain these, I do need EOL-at-EOF #vimuser 😉

Reply:
Ah sorry, I recently changed editor and forgot to set that up. Fixed :)